recipe/meta.yaml

```
@@ -522,7 +524,11 @@ requirements:
     - sysroot_linux-aarch64 {{ cudavars[major_minor]["sysroot_version"] }}  # [aarch64]
 {% endif %}
   run_constrained:
+{% if major_version < 11 %}
     - __cuda >={{ major_minor }}
+{% else %}
+    - __cuda >={{ major_version }}.0
```

Why is the `.0` needed? Couldn't this just be...

```suggestion
    - __cuda >={{ major_version }}
```

---

I think so, good point.. I've adjusted it!